index.html

```diff
@@ -8,7 +8,7 @@
     <title>Microverse Candidates Survey Form</title>
 </head>
 <body>
-  <div id="container">
+  <div id="container1">
```

Comment:
Kindly use better names for the containers. You have a variety of names you can use for your containers